compiler/src/dmd/cxxfrontend.d

```diff
@@ -419,6 +419,11 @@ void semanticTypeInfoMembers(StructDeclaration sd)
     return dmd.semantic3.semanticTypeInfoMembers(sd);
 }
 
+bool checkClosure(FuncDeclaration fd)
```

Comment:
I would have expected this to require an addition to `frontend.h`, apparently not.